extensions/interactions/EndExploration/directives/EndExplorationValidationServiceSpec.js

```diff
@@ -107,4 +107,26 @@ describe('EndExplorationValidationService', function() {
       currentState, customizationArguments, [], null);
     expect(warnings).toEqual([]);
   });
+
```

Comment:
This is the error I get on running the lint check locally for this file:  
```bash
 26:21  error  Unused injected value $rootScope   angular/di-unused
  26:21  error  Unused injected value $controller  angular/di-unused

✖ 2 problems (2 errors, 0 warnings)

```
Please remove these unused-dependencies.

Reply:
(For reference, please see [line 26](https://github.com/oppia/oppia/blob/develop/extensions/interactions/EndExploration/directives/EndExplorationValidationServiceSpec.js#L26) of the develop branch).